lib/gitsh/interactive_runner.rb

```diff
@@ -23,6 +26,7 @@ def run
       setup_readline
       handle_window_resize
       greet_user
+      load_gitshrc
```

Comment:
I think I prefer it after the greeting: if there are commands in the rc file that produce output, it makes sense if that output appears after the greeting.

For example, here's my current `.gitshrc` file:

```
:set color.ui true

:echo
status
:echo
log --oneline -n 1
```